src/test/suite/extensionFunctions.test.ts

```diff
@@ -9,9 +9,9 @@ import { expect } from 'chai';
 
 suite('Extension Functions Test Suite', () => {
 
-	const uriOne : string = 'https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/demos/markdown/didact-demo.didact.md';
-	const uriTwo : string = 'https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/demos/asciidoc/didact-demo.didact.adoc';
-	const uriThree : string = 'https://github.com/redhat-developer/vscode-didact/blob/master/demos/markdown/camelinaction/chapter1/cia2-chapter-1-v2.didact.md';
+	const uriOne = 'https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/demos/markdown/didact-demo.didact.md';
+	const uriTwo = 'https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/demos/asciidoc/didact-demo.didact.adoc';
+	const uriThree = 'https://github.com/redhat-developer/vscode-didact/blob/master/demos/markdown/camelinaction/chapter1/cia2-chapter-1-v2.didact.md';
```

Comment:
why the type is removed in this case?

Reply:
Because ESLint told me to remove it as an initializing with a string literal already makes the type definition obsolete.

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-inferrable-types.md